.solhint.json

```diff
@@ -14,6 +14,6 @@
         "no-empty-blocks": "off",
         "avoid-low-level-calls": "off",
         "gas-custom-errors": "off",
-        "imports-order": "warn"
+        "imports-order": "off"
```

Comment:
The `imports-order` lint doesn't work with disabled `no-unused-import`. This is an issue for Safe.sol and TestImports.sol.